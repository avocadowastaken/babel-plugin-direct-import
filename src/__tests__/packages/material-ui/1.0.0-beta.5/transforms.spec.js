const { runTransformSpec } = require("../../../../../tools/run-spec");

runTransformSpec(__dirname);
