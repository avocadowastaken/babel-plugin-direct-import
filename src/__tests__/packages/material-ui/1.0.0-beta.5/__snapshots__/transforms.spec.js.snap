// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Transform: material-ui@1.0.0-beta.5 should throw with: "unknown-module.js" 1`] = `"unknown: babel-plugin-direct-import: material-ui does not contain module \\"FlatButton\\""`;

exports[`Transform: material-ui@1.0.0-beta.5 should transform with: "each-module.js" 1`] = `
import {
  AppBar,
  Avatar,
  Badge,
  BottomNavigation,
  BottomNavigationButton,
  Button,
  ButtonBase,
  Card,
  CardActions,
  CardContent,
  CardHeader,
  CardMedia,
  Checkbox,
  Chip,
  CircularProgress,
  colors,
  createBreakpoints,
  createMuiTheme,
  createPalette,
  createTypography,
  Dialog,
  DialogActions,
  DialogContent,
  DialogContentText,
  DialogTitle,
  Divider,
  Drawer,
  FormControl,
  FormControlLabel,
  FormGroup,
  FormHelperText,
  FormLabel,
  Grid,
  Hidden,
  Icon,
  IconButton,
  Input,
  InputLabel,
  LinearProgress,
  List,
  ListItem,
  ListItemAvatar,
  ListItemIcon,
  ListItemSecondaryAction,
  ListItemText,
  ListSubheader,
  Menu,
  MenuItem,
  MenuList,
  MuiThemeProvider,
  Paper,
  Radio,
  RadioGroup,
  Snackbar,
  SnackbarContent,
  SvgIcon,
  Switch,
  Tab,
  Table,
  TableBody,
  TableCell,
  TableHead,
  TableRow,
  TableSortLabel,
  Tabs,
  TextField,
  Toolbar,
  Typography,
  withStyles,
  withTheme
} from "material-ui";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import AppBar from "material-ui/AppBar";
import Avatar from "material-ui/Avatar";
import Badge from "material-ui/Badge";
import BottomNavigation from "material-ui/BottomNavigation";
import { BottomNavigationButton } from "material-ui/BottomNavigation";
import Button from "material-ui/Button";
import ButtonBase from "material-ui/ButtonBase";
import Card from "material-ui/Card";
import { CardActions } from "material-ui/Card";
import { CardContent } from "material-ui/Card";
import { CardHeader } from "material-ui/Card";
import { CardMedia } from "material-ui/Card";
import Checkbox from "material-ui/Checkbox";
import Chip from "material-ui/Chip";
import { CircularProgress } from "material-ui/Progress";
import * as colors from "material-ui/colors";
import { createBreakpoints } from "material-ui/styles";
import { createMuiTheme } from "material-ui/styles";
import { createPalette } from "material-ui/styles";
import { createTypography } from "material-ui/styles";
import Dialog from "material-ui/Dialog";
import { DialogActions } from "material-ui/Dialog";
import { DialogContent } from "material-ui/Dialog";
import { DialogContentText } from "material-ui/Dialog";
import { DialogTitle } from "material-ui/Dialog";
import Divider from "material-ui/Divider";
import Drawer from "material-ui/Drawer";
import { FormControl } from "material-ui/Form";
import { FormControlLabel } from "material-ui/Form";
import { FormGroup } from "material-ui/Form";
import { FormHelperText } from "material-ui/Form";
import { FormLabel } from "material-ui/Form";
import Grid from "material-ui/Grid";
import Hidden from "material-ui/Hidden";
import Icon from "material-ui/Icon";
import IconButton from "material-ui/IconButton";
import Input from "material-ui/Input";
import { InputLabel } from "material-ui/Input";
import { LinearProgress } from "material-ui/Progress";
import List from "material-ui/List";
import { ListItem } from "material-ui/List";
import { ListItemAvatar } from "material-ui/List";
import { ListItemIcon } from "material-ui/List";
import { ListItemSecondaryAction } from "material-ui/List";
import { ListItemText } from "material-ui/List";
import { ListSubheader } from "material-ui/List";
import Menu from "material-ui/Menu";
import { MenuItem } from "material-ui/Menu";
import { MenuList } from "material-ui/Menu";
import { MuiThemeProvider } from "material-ui/styles";
import Paper from "material-ui/Paper";
import Radio from "material-ui/Radio";
import { RadioGroup } from "material-ui/Radio";
import Snackbar from "material-ui/Snackbar";
import { SnackbarContent } from "material-ui/Snackbar";
import SvgIcon from "material-ui/SvgIcon";
import Switch from "material-ui/Switch";
import { Tab } from "material-ui/Tabs";
import Table from "material-ui/Table";
import { TableBody } from "material-ui/Table";
import { TableCell } from "material-ui/Table";
import { TableHead } from "material-ui/Table";
import { TableRow } from "material-ui/Table";
import { TableSortLabel } from "material-ui/Table";
import Tabs from "material-ui/Tabs";
import TextField from "material-ui/TextField";
import Toolbar from "material-ui/Toolbar";
import Typography from "material-ui/Typography";
import { withStyles } from "material-ui/styles";
import { withTheme } from "material-ui/styles";
`;

exports[`Transform: material-ui@1.0.0-beta.5 should transform with: "empty-specifiers.js" 1`] = `
import "material-ui";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import "material-ui";
`;

exports[`Transform: material-ui@1.0.0-beta.5 should transform with: "local-name.js" 1`] = `
import {
  Button as MUIButton,
  Checkbox as MUICheckbox,
  TextField as MUITextField
} from "material-ui";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import MUIButton from "material-ui/Button";
import MUICheckbox from "material-ui/Checkbox";
import MUITextField from "material-ui/TextField";
`;

exports[`Transform: material-ui@1.0.0-beta.5 should transform with: "namespace-reexport.js" 1`] = `
import { colors } from "material-ui";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import * as colors from "material-ui/colors";
`;

exports[`Transform: material-ui@1.0.0-beta.5 should transform with: "namespace-specifier.js" 1`] = `
import * as all from "material-ui";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import * as all from "material-ui";
`;
